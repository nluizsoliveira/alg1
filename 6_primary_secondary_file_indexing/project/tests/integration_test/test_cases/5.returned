----------------------------------------------------------
Erro ao remover
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
10 - The Hunger Games - Suzanne Collins
----------------------------------------------------------
Registro removido
----------------------------------------------------------
NÃ£o encontrado
----------------------------------------------------------
TODO: Search for author
----------------------------------------------------------
Registro removido
----------------------------------------------------------
TODO: Search for author
