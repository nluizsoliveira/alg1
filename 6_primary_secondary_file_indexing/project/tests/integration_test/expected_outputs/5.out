----------------------------------------------------------
Erro ao remover
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
Registro inserido
----------------------------------------------------------
10 - The Hunger Games - Suzanne Collins
----------------------------------------------------------
Registro removido
----------------------------------------------------------
NÃ£o encontrado
----------------------------------------------------------
14 - Looking for Alaska - John Green
101 - An Abundance of Katherines - John Green
----------------------------------------------------------
Registro removido
----------------------------------------------------------
101 - An Abundance of Katherines - John Green
